docs/reference/api-conventions.asciidoc

```diff
@@ -660,15 +670,15 @@ string parameter.
 === URL-based access control
 
 Many users use a proxy with URL-based access control to secure access to
-Elasticsearch indices. For <<search-multi-search,multi-search>>,
+{es} data streams and indices. For <<search-multi-search,multi-search>>,
 <<docs-multi-get,multi-get>>, and <<docs-bulk,bulk>> requests, the user has
-the choice of specifying an index in the URL and on each individual request
+the choice of specifying a data stream or  index in the URL and on each individual request
 within the request body. This can make URL-based access control challenging.
 
-To prevent the user from overriding the index which has been specified in the
+To prevent the user from overriding the data stream or index which has been specified in the
 URL, add this setting to the `elasticsearch.yml` file:
 
     rest.action.multi.allow_explicit_index: false
```

Comment:
```suggestion
To prevent the user from overriding the data stream or index specified in the
URL, set `rest.action.multi.allow_explicit_index` to `false` in `elasticsearch.yml`.

```